app/views/episodes/_form_status.html.erb

```diff
@@ -1,81 +1,85 @@
-              <button type="button" class="btn btn-<%= episode_status_class(episode) %> dropdown-toggle dropdown-toggle-split border-start border-color-white" data-bs-toggle="dropdown" aria-expanded="false">
-                <span class="visually-hidden"><%= t(".toggle_dropdown") %></span>
-              </button>
-              <ul class="dropdown-menu">
-                <% if episode.published? && !episode.published_at_changed? %>
-                  <li><%= link_to t(".unpublish"), episode_path(episode, episode: {publishing_status: "draft"}), method: :put, class: "dropdown-item bg-danger text-white" %></li>
-                <% elsif policy(episode).destroy? %>
-                  <li>
-                    <button type="button" class="dropdown-item bg-danger text-white" data-bs-toggle="modal" data-bs-target="#confirm-destroy">
-                      <%= t(".destroy") %>
-                    </button>
-                  </li>
-                <% end %>
-              </ul>
-            <% end %>
+              <% if episode.published? || policy(episode).destroy? %>
+                <button type="button" class="btn btn-<%= episode_status_class(episode) %> dropdown-toggle dropdown-toggle-split border-start border-color-white" data-bs-toggle="dropdown" aria-expanded="false">
+                  <span class="visually-hidden"><%= t(".toggle_dropdown") %></span>
+                </button>
+                <ul class="dropdown-menu">
+                  <% if episode.published? && !episode.published_at_changed? %>
+                    <li><%= link_to t(".unpublish"), episode_path(episode, episode: {publishing_status: "draft"}), method: :put, class: "dropdown-item bg-danger text-white" %></li>
+                  <% elsif policy(episode).destroy? %>
+                    <li>
+                      <button type="button" class="dropdown-item bg-danger text-white" data-bs-toggle="modal" data-bs-target="#confirm-destroy">
+                        <%= t(".destroy") %>
+                      </button>
+                    </li>
+                  <% end %>
+                </ul>
+              <% end %>
+            </div>
           </div>
-        </div>
-      <% end %>
+        <% end %>
 
-    <% else %>
-      <p class="status-text"><strong><%= t(".new_hint") %></strong></p>
+      <% else %>
+        <p class="status-text"><strong><%= t(".new_hint") %></strong></p>
 
-      <% if policy(episode).create? %>
-        <% data = {disable_with: t(".creating"), upload_with: t(".uploading")} %>
-        <%= form.submit t(".create"), class: "btn btn-warning", data: data %>
+        <% if policy(episode).create? %>
+          <% data = {disable_with: t(".creating"), upload_with: t(".uploading")} %>
+          <%= form.submit t(".create"), class: "btn btn-warning", data: data %>
+        <% end %>
       <% end %>
-    <% end %>
+    </div>
   </div>
+
+  <%= render "form_distribution", episode: episode, form: form %>
```

Comment:
This feels a little weird to call it from inside the status box. But I suppose the modals work?